src/components/general/NavBanner.js

```diff
@@ -29,15 +29,14 @@ const NavBanner = props => {
           })}
         >
           <div className={styles.tabs}>{props.tabs}</div>
-          <div className={styles.dropdown}>
-            {props.dropdownDisabled ? null : (
-              <Dropdown
-                variant='border'
-                menu={menu}
-                disabled={props.dropdownDisabled}
-              />
-            )}
-          </div>
+
+          {props.isArtist && (
+            <div className={styles.dropdown}>
+              {!props.dropdownDisabled && (
```

Comment:
Nice simplification! Do we want to still render the outer dropdown container if dropdownDisabled?

Reply:
This is a great question that also had me wondering. We can circle back to it and merge the conditions if needed. Would be cleaner for sure as long as it makes sense functionally :)